spec/beaker/host/cisco_spec.rb

```diff
@@ -14,10 +14,10 @@ module Cisco
 
     describe '#prepend_commands' do
 
-      context 'for cisco-5' do
+      context 'for cisco-nexus' do
 
         before :each do
-          @platform = 'cisco-5-x86'
+          @platform = 'cisco-nexus-x86'
```

Comment:
Just trying to avoid another breaking change in the future (`cisco-nexus-x86_64` ==> `cisco_nexus-VERSION-x86_64`). If that's not a concern from Puppet's side of things, fine by me.